### PR TITLE
STL-2208 Creating the Pension Expenses Information subpage

### PR DESCRIPTION
**Pull request by hamo225**
# Short Description
- Create a subpage with information on Pension Expenses that will carry information for users and links to the other subpages.

# Changes
- adds routing for the new page using the new `render_react_content_page_template`. 
- Separates out the general styles that can be used by other content pages.
- adds media query for the `infoBox` component for smaller screens. 
- adds the text to `translations.js`
- Create the `PensionExpensesInfoPage`
- adds test for the `PensionExpensesInfoPage`

# Feedback
- Any? 

# How to review this Pull Request
[Link to Confluence](https://digitalservice4germany.atlassian.net/wiki/spaces/STL/pages/117637157/Development+Process#Approve-or-comment) (internal)

### CODE REVIEW

File: webapp/app/routes.py [hunk 1]
```diff
@@ -352,6 +352,13 @@ def ambassadorMaterial():
             props=AmbassadorInfoMaterialProps(plausible_domain=Config.PLAUSIBLE_DOMAIN).camelized_dict(), 
             component='AmbassadorInfoMaterialPage')
 
+    @app.route('/vorsorgeaufwendungen', methods=['GET'])
+    @add_caching_headers
+    def PensionExpensesInfoPage():
+        return render_react_content_page_template(
+             props=PensionExpensesProps().camelized_dict(),
```
Comment (SannyNguyenHung) [inline webapp/app/routes.py:359]: please indent properly :)

File: webapp/client/src/components/ContentPagesGeneralStyling.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import styled from "styled-components";
+
+export const ContentWrapper = styled.div`
+  padding-left: var(--spacing-03);
+  padding-right: var(--spacing-03);
+  margin: 0 auto;
+  max-width: var(--main-max-width);
+`;
+export const List = styled.ul`
```
Comment (SannyNguyenHung) [inline webapp/client/src/components/ContentPagesGeneralStyling.js:9]: Append newline on line 8

**Review by SannyNguyenHung — COMMENTED** [review]
Can you please add a new page to our story book? :)

File: webapp/client/src/pages/PensionExpensesInfoPage.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import InfoBox from "../components/InfoBox";
+// eslint-disable-next-line import/named
+import StepHeaderButtons from "../components/StepHeaderButtons";
+import { anchorBack, anchorList } from "../lib/contentPagesAnchors";
```
Comment (punknoir101) [inline webapp/client/src/pages/PensionExpensesInfoPage.js:6]: line 4 hast to be above line 6

File: webapp/client/src/components/ContentPagesGeneralStyling.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+export default {
+  ContentWrapper,
+  List,
+  AnchorListItem,
+  ListItem,
+  Headline1,
+  Headline2,
+  Paragraph,
+};
```
Comment (punknoir101) [inline webapp/client/src/components/ContentPagesGeneralStyling.js:70]: After feedback from UX we have to adjust some general spaces and sizes; I will go to adjust it after this PR.

File: webapp/client/src/components/InfoBox.js [hunk 5]
```diff
@@ -48,6 +48,10 @@ const Figure = styled.figure`
   img {
     max-width: 500px;
     height: auto;
+
+    @media (max-width: 425px) {
+      max-width: 100%;
+    }
   }
```
Comment (punknoir101) [inline webapp/client/src/components/InfoBox.js:55]: We are going to optimize the image in another subtask, but leave it as it is right now.

**Review by punknoir101 — APPROVED** [review]

**Review by punknoir101 — COMMENTED** [review]
You have to add the url path to contentPagesAnchors.js in lib folder

File: webapp/client/src/components/ContentPagesGeneralStyling.js (hunk 4 above)
Comment (punknoir101) [reply webapp/client/src/components/ContentPagesGeneralStyling.js:70]: We could also provide them as "real" components.

**Pull request merged**